### PR TITLE
Update using-the-support-portal.md

### PR DESCRIPTION
**Pull request by ThomasLehnerSpryker**
added case assistant description

## PR Description

TBD

## Checklist
- [x ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/about/all/support/using-the-support-portal.md [hunk 1]
```diff
@@ -17,44 +17,67 @@ This document describes how to use the support portal to create and manage your
 
 ## Prerequisites
 
-You already have a customer login provisioned for you. If you do not have an account yet, request one using this form.
+You should have a customer account provisioned for you. If you don't, request one using this form.
```
Comment (andriitserkovnyi) [inline docs/about/all/support/using-the-support-portal.md:20]: Link missing

File: docs/about/all/support/using-the-support-portal.md [hunk 2]
```diff
@@ -17,44 +17,67 @@ This document describes how to use the support portal to create and manage your
...
+Case Assistant guides you through the process of creating your requests by asking questions making sure they are routed properly.
 
-Requests can be questions or change requests that we do not yet offer as categories in *Hosting Change Requests*.
+The following are the top-level categories of cases and their description.
 
-### Problems
+### Report a Problem or Incident
 
-Problems can be reports suspecting a bug or issues with our hosting. Please make sure that you follow this guide to include everything we need to get working on your case right away.
+Problems can be reports suspecting a bug or issues with the hosting services. To speed up the resolution of such cases, make sure to follow this document before you submit your request.
```
Comment (andriitserkovnyi) [inline docs/about/all/support/using-the-support-portal.md:33]: Links missing

File: docs/about/all/support/using-the-support-portal.md (hunk 1 above)
Comment (ThomasLehnerSpryker) [reply docs/about/all/support/using-the-support-portal.md]: link added

File: docs/about/all/support/using-the-support-portal.md (hunk 2 above)
Comment (ThomasLehnerSpryker) [reply docs/about/all/support/using-the-support-portal.md]: link added

**Pull request merged**